_posts/2016-05-20-drupalcon-nola-recap.md

```diff
@@ -0,0 +1,118 @@
+
+  My last session of Drupalcon was Dries's [State of Drupal](https://events.drupal.org/neworleans2016/sessions/state-drupal). Since I missed the [#Driesnote](https://twitter.com/hashtag/DriesNote?src=hash) in person (just finished watching it), and contributed to [the survey](http://buytaert.net/state-of-drupal-2016-survey) he put out, and [commented on his blog post](http://buytaert.net/how-is-drupal-8-doing#comment-125541) about the adoption of Drupal 8, I figured I till needed more. Sure enough, much of the material I was aware of at the time, but as I am wont to do, I was the [first to ask Dries a question](https://youtu.be/Ct0IYLgPYos?t=1923) (admittedly a bit circuitous and verbose) as to how to improve the experience of trying to contribute to Drupal. [XJM](https://www.drupal.org/u/xjm) rushes to the mic to give a great answer, which ended up driving my involvement with [Jeremy Thorson](https://www.drupal.org/u/jthorson) (who also speaks thereafter) at the sprint the following day.
+
+The day concluded with the marching memorial putting Drupal 6 to rest. I observed the wonderful jazz band that accompanied it, but passed up the opportunity to head to the Rusty Nail for the 3rd time in as many days. Kosta nd Lisa were in the air and on their way home at this point.
+
+### Friday - Code sprinting / Heading home
+
+The Friday sprints were surprisingly well attended. Sprint lounges were occupied all week, and some sprinters promised to continue throughout the weekend at the convention center. This dedication to the project is impressive and inspirational (and I've argued necessary) to the future of Drupal's success.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Putting my 💰 where my 🤑 is and investing in <a href="https://twitter.com/hashtag/Drupal?src=hash">#Drupal</a>:  <a href="https://t.co/s1aUuoKxkv">https://t.co/s1aUuoKxkv</a> <a href="https://twitter.com/DrupalConNA">@DrupalConNA</a> code sprinting 🏃 ! <a href="https://t.co/mlSXsZif4m">pic.twitter.com/mlSXsZif4m</a></p>&mdash; Savas Labs (@Savas_Labs) <a href="https://twitter.com/Savas_Labs/status/731148211218436097">May 13, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+It is in the track world just as it is in the coding world: **sprinting is hard!**. It is difficult to get organized, get oriented (or help orient) and contribute effectively to a global project in a narrow window of time.
+
+Given my newfound desire to ease the barriers and access to contributing to the project in general, I decided to sprint to make the drupal.org contributor experience better. In doing so, I got [my very own installation of the drupal.org website](https://chrisarusso-drupal.dev.devdrupal.org/) of which I had complete control. **Fun**!
```

Comment:
https://chrisarusso-drupal.dev.devdrupal.org/ is password protected, as I assume it should be. Just wanted to point that out, it may not be worth linking to though.